hash/gbcolor.xml

```diff
@@ -28397,4 +28397,59 @@ license:CC0-1.0
 			</dataarea>
 		</part>
 	</software>
+
+
+<!-- Aftermarket games -->
+
```

Comment:
This comment doesn’t even make sense – “aftermarket” refers to modifications made after purchase.

Reply:
..That would be a patch. "Aftermarket" is the common term for those games released after a system's lifetime.

Reply:
Which term do you suggest in its place?

Reply:
I mean, we also can't just classify these as "homebrew", having been commisioned by a multi-billion dollar company.

Reply:
I think the word we were looking for was "Freeware".